src/braket/circuits/angled_gate.py

```diff
@@ -96,4 +107,5 @@ def __eq__(self, other):
         return False
 
     def __repr__(self):
-        return f"{self.name}('angle': {self.angle}, 'qubit_count': {self.qubit_count})"
+        dagger_str = "^†" if self._adjoint else ""
```

Comment:
You missed this ^†

Reply:
How come it wasn't caught by unit tests?

Reply:
Ah, coverage doesn't seem to cover branches within an expression. I'll fix this.